src/blockfilter.cpp

```diff
@@ -233,3 +233,24 @@ BlockFilter::BlockFilter(BlockFilterType filter_type, const CBlock& block, const
         throw std::invalid_argument("unknown filter_type");
     }
 }
+
+uint256 BlockFilter::GetHash() const
```

Comment:
In commit "blockfilter: Serialization methods on BlockFilter." (c14e4d5a950ed40d96a8956e4622e1d9670da72d)

Note: https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki#filter-headers is the relevant section of the bip describing block filter hash (`GetHash`) and filter header (`ComputeHeader`) derivations.